src/styles/base/_type.scss

```diff
@@ -78,18 +78,6 @@ $fs-small: (
   }
```

Comment:
We have the first half of this file in mixins/type in the tokens repo. Do we still need it here?

Reply:
We would need to keep the display variables since those aren't in tokens and we use them in the docs site. Do we want to add those to tokens? Or keep the display variables scoped to docs?